scrapers/Puffynetwork.yml

```diff
@@ -0,0 +1,39 @@
+name: puffynetwork
+sceneByURL:
+  - action: scrapeXPath
+    url: 
+      - puffynetwork.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    common:
+      $performer: //dl/dd/a
+    scene:
+      Title: 
+        selector: //h2[@class="title"]/span/text()
+        replace:
+          - regex: ^(.*)[—]\s+
+            with:
+      Date:
+        selector: //dl/dt[2]/span/text()
+        parseDate: Jan 2, 2006
+      Details: //div[@class="show_more"]/text()[1]|//div[@class="show_more showed_more"]/text()[1]

```

Comment:
`Details: //div[@class="show_more"]/text()[1]`
is enough , the second div doesn't exist in the page source
When viewing the scene from firefox if you click on the more button the `show more` div becomes `show_more showed_more` 
Since our scraper takes the page source directly and no js is involved this isn't needed